data_inconsistencies.py

```diff
@@ -0,0 +1,54 @@
+import os
+import csv
+
+
+def get_inconsistencies():
+    files = os.listdir('./data/')
+
+    parsed = {}
+    for file in files:
```

Comment:
Can you rename the `file` variable? It's a keyword in Python, so it gets highlighted differently than other variables (which can be confusing).